CONTRIBUTING.md

```diff
@@ -2,6 +2,15 @@
 The SPRAS maintainers welcome external contributions.
 Code contributions will be licensed using the project's [MIT license](LICENSE).
 
+If you are looking for the pathway reconstruction guide,
+go to the [Contributing a new pathway reconstruction guide](#contributing-a-new-pathway-reconstruction-guide) header.
+
+## On `.test_durations`
```

Comment:
I would move this much lower. Most of our contributing guide readers will be new SPRAS team members who use it to learn SPRAS. I agree that `.test_durations` needs to be documented and this is a good place to do it, but that could be toward the end where we explain pre-commit.